### DISCUSSION TITLE
Does this work with MacOS?

### DISCUSSION THREAD

**Discussion by vietanhle0101** [kul-optec/alpaqa#2] [Q&A]
Hi, great project, does this work with MacOS?

**Comment by tttapa** [comment]
Hi,

The [C++ installation instructions](https://kul-optec.github.io/alpaqa/develop/Doxygen/installation.html) should work for macOS , with the caveat that your default compiler might be too old for some optional components, but this should not be an issue. 

The pre-built Python packages support macOS 12 and later (both Intel and ARM), starting from the 1.0.0 pre-releases. You can install the Python package using `python3 -m pip install --upgrade --pre alpaqa`.
